### ISSUE TITLE
androidMainActivityKotlinLines2WithStatusBar et al not applied in MainActivity.kt

### ISSUE THREAD

**Issue by TomTom101** [henriquearthur/flutter_native_splash#56]
The default MainActivity created by `flutter create splash_test` looks like this:

```kotlin
package com.example.splash_test

import androidx.annotation.NonNull;
import io.flutter.embedding.android.FlutterActivity
import io.flutter.embedding.engine.FlutterEngine
import io.flutter.plugins.GeneratedPluginRegistrant

class MainActivity: FlutterActivity() {
    override fun configureFlutterEngine(@NonNull flutterEngine: FlutterEngine) {
        GeneratedPluginRegistrant.registerWith(flutterEngine);
    }
}
```

The Kotlin reference line to include the code that shows the status bar after is was hidden for the splash screen expects the non-existant line `super.onCreate(savedInstanceState)'`.

**Comment by TomTom101** [comment]
Duplicate of #35

**Comment by edufolly** [comment]
Probably this is the solution: [#54](https://github.com/henriquearthur/flutter_native_splash/issues/54#issuecomment-589237135)

**Comment by TomTom101** [comment]
The actual problem observed here is that when setting windowFullscreen: true in order to show the splash screen without status bar, the status bar does not come back when the app is starting. It did however solve that flashy black screen issue. Still not sure what to do to get a full screen splash screen and an app w/ status bar.

**Comment by ademonte** [comment]
Hello,

I had to add this in the `MainActivity.kt` file to make the status bar visible after the splash screen

```kotlin
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Our native splash screen was FULLSCREEN, let's clear the flag.
        window.clearFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN)
    }
```

**Comment by TomTom101** [comment]
Thanks @ademonte 
This doesn't compile for me:
```e:  /app/MainActivity.kt: (15, 5): 'onCreate' overrides nothing
e: /app/MainActivity.kt: (15, 47): Unresolved reference: Bundle
```

**Comment by ademonte** [comment]
yes of course

```kotlin
package com.hospimedia.hospimediaapp

import android.os.Bundle
import androidx.annotation.NonNull;
import android.view.WindowManager;
import io.flutter.embedding.android.FlutterActivity
import io.flutter.embedding.engine.FlutterEngine
import io.flutter.plugins.GeneratedPluginRegistrant

class MainActivity: FlutterActivity() {
    override fun configureFlutterEngine(@NonNull flutterEngine: FlutterEngine) {
        GeneratedPluginRegistrant.registerWith(flutterEngine);
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Our native splash screen was FULLSCREEN, let's clear the flag.
        window.clearFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN)
    }
}
```

**Comment by vtisnado** [comment]
I tested the solution provided by @ademonte (Thank you!), but I have to mention that you should  add that code after running `flutter pub run flutter_native_splash:create` 

If you add the code before, for any reason I don't know a boilerplate code is generated in `MainActivity.kt` and the build fails.

**Comment by jonbhanson** [comment]
Should be resolved by 48981828e831768c4ad89a35475e8f9f24e0aca5.
